src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/NotificationsService.cs

```diff
@@ -30,17 +28,17 @@ public async Task NotifySubscribersIfApplicable(params Guid[] releaseIds)
         {
             var releasesToNotify = _context.Releases
                 .Include(r => r.ReleaseStatuses)
+                .Include(r => r.Updates) // used by BuildPublicationNotificationMessage
+                .Include(r => r.Publication) // used by BuildPublicationNotificationMessage
```

Comment:
I don't think this set of includes necessarily generates the most efficient query. It would be interesting to see what this is if there's a lot of status updates and update notes.

I think it will join release, updates, and publication and release statuses so it pulls back all the fields of the publication for every update / release status combination and if there's a lot of approvals and update notes that would be pulling back a lot of rows with a lot of duplicated data!

Reply:
I figured I'd do these includes here and save them being done again in `BuildPublicationNotificationMessage`. I'll remove them and let `BuildPublicationNotificationMessage` handle it.